ssl/statem/statem_lib.c

```diff
@@ -1036,11 +1046,14 @@ WORK_STATE tls_finish_handshake(SSL *s, WORK_STATE wst, int clearbufs)
              */
             if (!SSL_IS_TLS13(s))
                 ssl_update_cache(s, SSL_SESS_CACHE_CLIENT);
-            if (s->hit)
-                s->ctx->stats.sess_hit++;
+            if (s->hit) {
+                CRYPTO_atomic_add(&s->session_ctx->stats.sess_hit, 1, &discard,
+                                  s->session_ctx->lock);
+            }
```

Comment:
No reason for the {}

Reply:
I must have misremembered the style rule about single-line vs. single-statement conditional bodies.